guidelines/python-cli-ux-guidelines.md

```diff
@@ -0,0 +1,44 @@
+# Python CLI UX Guidelines
```

Comment:
why not just put this as a subsection in the CONTRIBUTING.md file?

perhaps the images could live in a new `assets/` folder like we have [in secretless](https://github.com/cyberark/secretless-broker/tree/master/assets)?

Reply:
I moved the images to the new assets folder as suggested. I think the UX guidelines are a bit too long for  CONTRIBUTING.md so I added a new section in the  CONTRIBUTING.md doc and linked to the guidelines from there